h/static/styles/partials-v2/_share-widget.scss

```diff
@@ -0,0 +1,152 @@
+// Share widget
+// ---------------
+// The widget that allows social linking and the ability to
+// copy a url for use elsewhere.
+
+
+// container for widget should not be displayed
+.js-share-widget {
```

Comment:
we are explicitly using visibility so that we can easily calculate height and width without displaying the element. With display:none, getBoundingClientRect returns 0. That utility uses display none so it's not helpful here